dlrover/python/diagnose/common/diagnose_data.py

```diff
@@ -13,8 +13,7 @@
 
 from abc import ABCMeta, abstractmethod
 from datetime import datetime
-
-from dlrover.python.util.file_util import read_last_n_lines
+from typing import List, Optional
```

Comment:
```diagnose_data.py``` should be named as ```diagnosis_data.py```

Reply:
Also, there are 2 'diagnose_data.py' in common and master package. Need to unified.